README.md

```diff
@@ -0,0 +1,57 @@
+# ami-builder-configs
+Configuration files for building various AMIs using ami-builder
+
+Each time ami-builder-config code has a new commit & is merged, a new GitHub release is created and zipped
+This gets picked up by Concourse pipeline job 'Ami-builder-release' which is monitoring every hour looking for a new release.
```

Comment:
We could link to the pipeline in Concourse here?

Reply:
done